src/goesproc/goesproc.cc

```diff
@@ -87,7 +88,9 @@ int main(int argc, char** argv) {
         std::unique_ptr<Handler>(
           new EMWINHandler(handler, fileWriter)));
     } else if (handler.type == "dcs") {
```

Comment:
It would be useful to also update `etc/goesproc.conf` to add an example DCS handler entry in the file.